src/integrationTest/groovy/wooga/gradle/snyk/tasks/SnykTestBaseIntegrationSpec.groovy

```diff
@@ -45,14 +44,80 @@ abstract class SnykTestBaseIntegrationSpec<T extends SnykTask> extends SnykCheck
         """.stripIndent()
     }
 
+    def setSnykPolicy(File policy = null) {
+        appendToSubjectTask("""
+        policyPath = ${wrapValueBasedOnType(policy ?: snykPolicy, File)}
+        """.stripIndent())
+    }
+
+    File snykPolicy
+
+    def setup() {
```

Comment:
I would add a comment here to explain why we setting it so, and perhaps add a TODO to remind us to remove this once snyk fixes the bug upstream?

Reply:
This actually doesn't do anything except create an empty policy file. In a test (sure we don't have this yet) I can use this file to add custom policies for a test. So it is used as a workaround for the issue with snyk but not actually this code. There are 3 pieces.
* the variable to hold a snykPolicy
* the code to assign a policy file (also generic)
* and the code to specificly use this in the test which has a comment why it is there.

Reply:
You are right; I should have instead said that I would like us to document why we are setting a custom policy file somewhere in the integration tests.

Reply:
I added a comment in the specific test that executes said method